### PR TITLE
Support/wagtail 5.1

### PR DESCRIPTION
**Pull request by nickmoreton**
# Wagtail 5.1 update

There are no changes to the working code, just changes to add 5.1 to the test matrix.

I also made an adjustment in the test settings, see the comment next to it. There may be a better way to do that?

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [tox]
 envlist =
-    py{38,39,310}-dj{32,41}-wagtail{41,42}
-    py{310,311}-dj{41}-wagtail{41}
-    py{310,311}-dj{42}-wagtail{50,main}
+    py{38,39,310}-dj{32,41}-wagtail{41,42,50,51}
```
Comment (zerolab) [inline tox.ini:3]:
```suggestion
    py{38,39,310}-dj{32,41}-wagtail{41,50,51}
```

File: tox.ini [hunk 2]
```diff
@@ -24,7 +24,8 @@ deps =
     dj42: Django>=4.2,<4.3
     wagtail41: wagtail>=4.1,<4.2
     wagtail42: wagtail>=4.2,<5.0
```
Comment (zerolab) [inline tox.ini:26]: This can be removed too

**Review by zerolab — COMMENTED** [review]
Left a couple of drive-by notes.

**Comment by katdom13** [comment]
Superseded by: https://github.com/torchbox/wagtail-storages/pull/42

**Pull request closed without merging**